vaadin-crud-flow-parent/vaadin-crud-testbench/src/main/java/com/vaadin/flow/component/crud/testbench/CrudElement.java

```diff
@@ -81,7 +83,12 @@ public void openRowForEditing(int row) {
         if (isEditOnClick()) {
             this.getGrid().getCell(row, 0).click();
         } else {
-            this.$("vaadin-crud-edit").all().get(row).click();
+            GridTHTDElement editCell = getGrid().getAllColumns().stream()
+                    .map(column -> getGrid().getRow(row).getCell(column))
```

Comment:
Not sure if this is an premature optimization, but doesn't it make sense to store `getGrid().getRow(row)` in a variable?

Reply:
Done it this way, so it will be obvious what's wrong if you are getting `NoSuchElementException`.

Reply:
Updated now.